src/main/features/eligibility/views/macro/mcolSupport.njk

```diff
@@ -0,0 +1,9 @@
+{% macro mcolSupportWidget(label = 'Help and advice') %}
+  <h2 class="heading-medium">{{ t(label) }}</h2>
+  <pre>{{ t('Money Claim Online helpdesk') }}</pre>
+  <pre>{{ t('Email: ccbc@hmcts.gsi.gov.uk') }}</pre>
+  <pre>{{ t('Telephone: 0300 123 1057') }}</pre>
+  <pre>{{ t('Monday to Friday, 8:30am to 5pm') }}</pre>
+  <p><a target="_blank" href="https://www.gov.uk/call-charges">{{ t('Find out about call charges') }}
```

Comment:
A `Find out about call charges` link could be separate macro as it is now used at least in two places.

Reply:
do you think this macro should be in common-frontend?

Reply:
done

Reply:
created macro in common-frontend for callCharges